docs/source/architecture.rst

```diff
@@ -186,6 +186,6 @@ which is used for serving `reveal.js`_ HTML slideshows.
 .. links:
 
 .. _jinja: http://jinja.pocoo.org/
-.. _filter: http://jinja.pocoo.org/docs/dev/templates/#filters
```

Comment:
Hmm still seems to be complaining about pocoo.org fetches -- maybe the line before or somewhere else in the project we're fetching objects that don't exist anymore?

Reply:
yes @MSeal ! i'm really confused about the CI issue. Here is what i know. 

that url - jinja.pocoo.org/docs resolves to https://jinja.palletsprojects.com/en/3.0.x/

i've definitely seen builds have a hard time with redirects. i just fixed that but i'm still seeing that CI issue. i suspect somewhere there are other pocoo.org references perhaps? i can search for that next. 

http://jinja.pocoo.org/docs/dev/objects.inv 

i think i fixed it. the link for intersphinx_mapping was the old url. give me a few minutes to push it up. fingers crossed :)

Reply:
fixed this URL to be the new jinja docs link. palletsprojects.com